src/script/run_tox.sh

```diff
@@ -15,6 +15,17 @@ script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 : ${WITH_PYTHON2:=$(get_cmake_variable WITH_PYTHON2)}
 : ${WITH_PYTHON3:=$(get_cmake_variable WITH_PYTHON3)}
 
+
```

Comment:
might want to drop the extra empty lines.

Reply:
also the commit message is kind of vague when one is also looking at 453af5c1b3d964add347d9d10f52d4f9e402d85d. it's more about integrating the tox test into cmake.

Reply:
fixed and fixed